pynav/filters.py

```diff
@@ -100,28 +98,21 @@ def predict(
         if x_jac is None:
             x_jac = x.state
 
-        # Use current input provided, or the one previously.
-        if use_last_input:
-            u_eval = self._u
-        else:
-            u_eval = u
-
-        if u_eval is not None:
-            A = self.process_model.jacobian(x_jac, u_eval, dt)
-            Q = self.process_model.covariance(x_jac, u_eval, dt)
-            x.state = self.process_model.evaluate(x.state, u_eval, dt)
+        if u is not None:
```

Comment:
Wait so what does happen if u is None? Just no prediction? Confused about line 101 if statement.

Reply:
yup, just nothing and returns the same state. i think this was there to deal with estimator startup in the previous framework, but now might not be necessary